src/app/data-planes/components/PolicyTypeEntryList.vue

```diff
@@ -100,15 +100,26 @@
 </template>
 
 <script lang="ts" setup>
-import { KTable } from '@kong/kongponents'
-import { PropType } from 'vue'
-
 import AccordionItem from '@/app/common/AccordionItem.vue'
 import AccordionList from '@/app/common/AccordionList.vue'
 import CodeBlock from '@/app/common/CodeBlock.vue'
 import PolicyTypeTag from '@/app/common/PolicyTypeTag.vue'
 import TagList from '@/app/common/TagList.vue'
-import { PolicyTypeEntry, TableHeader } from '@/types/index'
+import type { LabelValue, TableHeader } from '@/types/index.d'
+import type { RouteLocationNamedRaw } from 'vue-router'
+
+export type PolicyTypeEntryConnection = {
+  sourceTags: LabelValue[]
+  destinationTags: LabelValue[]
+  name: string | null
+  origins: Array<{ name: string, route: RouteLocationNamedRaw }>
+  config: string | null
```

Comment:
I know we haven't got to moving policy type things over to our data layer yet, but is there a straight forward alternative to `null`? If not don't worry this will no doubt be changed pretty soon.